core/lib/dr_tools.h

```diff
@@ -341,6 +341,12 @@ typedef enum {
      * \note Windows only.
      */
     DR_MEMORY_DUMP_LDMP = 0x0001,
+    /**
+     * Memory dump in Executable and Linkable Format.
+     *
+     * \note X64 Linux only.
+     */
+    DR_MEMORY_DUMP_ELF = 0x0002,
```

Comment:
I don't see the limitations clause for when this can be called?

Reply:
I've added a comment above. PTAL.